src/test.spec.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { sqlParserFactory } from './parser/druidsql';
+import { FUNCTIONS } from './test-utils';
+
+const parser = sqlParserFactory(FUNCTIONS);
+
+describe.skip('test', () => {
```

Comment:
It's a garbage file I use as like a testing area during development to make sure things are working. Having it in one file is nice because you can make changes then run jest test -u and look at an individual change. I will make it un tracked though since it is not important at all, I will also rename it.